### DISCUSSION TITLE
v3.11.0

### DISCUSSION THREAD

**Discussion by janzandr** [EnMAP-Box/enmap-box#173] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/EnMAP-Box/enmap-box/releases/tag/v3.11.0'>v3.11.0</a>.</em>

**Comment by jakimowb** [comment]
Do you like to create a new discussion for each new bug release, i.e. 'v3.11.1', 'v3.11.2' etc.? I recommend to have just one for the current release that is prepared / active

**Comment by janzandr** [comment]
I don't know. Probably nobody will use it anyways :-)
